pyproject.toml

```diff
@@ -33,6 +33,9 @@ fastapi = "^0.95.0"
 uvicorn = "^0.21.1"
 psutil = "^5.9.4"
 joblib = "^1.2.0"
+virtualenv = "^20.23.0"
+dockerfile = "^3.2.0"
+watchfiles = "^0.19.0"
```

Comment:
I imagine these libraries are pretty lightweight.

Reply:
Looking at pypi:

watchfiles: 36.0kB
virtualenv 3.3MB
dockerfile: 1.8MB
yaspin: 18.5kB

I think it's fine for now. I think the requirements will end up getting lighter once we split up into the various parts of the package.